ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java

```diff
@@ -291,14 +294,13 @@ private void validateTopicProperties(final String topic,
     if (topicDescription.partitions().size() != numPartitions
         || topicDescription.partitions().get(0).replicas().size() < replicationFactor) {
       throw new KafkaTopicException(String.format(
-              "A Kafka topic with the name '%s' already exists, with different partition/replica "
-              + "configuration than required. KSQL expects %d partitions (topic has %d), and %d "
-              + "replication factor (topic has %d).",
-              topic,
-              numPartitions,
-              topicDescription.partitions().size(),
-              replicationFactor,
-              topicDescription.partitions().get(0).replicas().size()
+          "Topic '%s' does not conform to the requirements Partitions:%d v %d. Replication: %d "
```

Comment:
Is the error message change intentional? Looks like it might be a bad merge...

Reply:
I don't know, I didn't touch that code

Reply:
Then something weird went on with a merge - can you add back in the changed error message please, i.e. there should be no difference here in your PR.

Reply:
Then something weird went on with a merge - can you add back in the changed error message please, i.e. there should be no difference here in your PR.

Reply:
@big-andy-coates Ok